src/common/entities.ts

```diff
@@ -1,3 +1,6 @@
+import { ColumnDef } from "@tanstack/react-table";
+import { GridTrackSize } from "config/entities";
```

Comment:
@MillenniumFalconMechanic apps that use `findable-ui` needs these imports to be relative here and elsewhere (if there are any).